src/pages/EditShelterSupply/EditShelterSupply.tsx

```diff
@@ -173,42 +174,42 @@ const EditShelterSupply = () => {
           {...modalData}
         />
       )}
-      <div className="flex flex-col h-screen items-center">
+      <div className="flex flex-col h-screen max-w-full items-center">
         <Header
           title="Editar Itens"
-          className="bg-white [&_*]:text-zinc-800 border-b-[1px] border-b-border"
           startAdornment={
             <Button
               variant="ghost"
-              className="[&_svg]:stroke-blue-500"
               onClick={() => navigate(`/abrigo/${shelterId}`)}
             >
               <ChevronLeft size={20} />
             </Button>
           }
         />
-        <div className="p-4 flex flex-col max-w-5xl w-full gap-3 items-start">
+        <div className="p-4 flex flex-col max-w-5xl w-full gap-3 items-start pt-20">
           <h6 className="text-2xl font-semibold">Editar itens do abrigo</h6>
           <p className="text-muted-foreground">
-            Antes de adicionar um novo item, confira na busca abaixo se ele já não foi cadastrado.
+            Antes de adicionar um novo item, confira na busca abaixo se ele já
+            não foi cadastrado.
```

Comment:
Minha sugestão é deixar esse texto mais afirmativo:
```suggestion
            Antes de adicionar um novo item, confira na busca abaixo se ele já
            está cadastrado.
```